src/app/CommandHandler.cpp

```diff
@@ -32,6 +32,7 @@
 #include <app/RequiredPrivilege.h>
```

Comment:
Adding blocking comment that I will resolve when I am online for the next few hours